### DISCUSSION TITLE
DWMBlurGlass doesn't run itself by itself

### DISCUSSION THREAD

**Discussion by daveo76** [Maplespe/DWMBlurGlass#161] [Q&A]
When I start my PC. Most times DWMBlurGlass doesn't run by itself. I'm having to open the program and uninstall it. Then install it. To make it run.
Using DWMBlurGlass v1.0.4 and Windows 11 23H2 Build 22631.3085

**Comment by Maplespe** [comment]
Please check if your scheduled task service is working properly.
Make sure that you can find the "DWMBlurGlass_Extend" item in your computer's scheduled task manager and that it is not blocked by other programs.

**Comment by daveo76** [comment]
I've checked the Task Scheduler. It was already on it list and enabled. I even added it to the Malwarebytes allow list.  (That what I use). I also disbled malwarebytes and Windows security. No difference. Now when I try to run it manually I get a error message "Failed to load component! Error messge: VirtualAllocEx failed!". Perhaps it something to do with Windows 11 23H2 22631.3085

**Comment by Maplespe** [comment]
This seems to be the same as #83, do you still have SecureUXTheme installed?

**Comment by daveo76** [comment]
No I don't have any other themes installed. I found out it doesn't seem to like explorer patcher. When I uninstall that. Then DWMBlurGlass runs smoothly. But I like explorerpatcher. Which enables blur on taskbar and change it to windows 10 taskbar

**Comment by daveo76** [comment]
Seems to be working fine. Since upgraded to 2.0.1. This can be marked as completed

**Reply by daveo76** [reply]
It gone back to not auto starting from boot up. Using Windows 11 Pro 23H2 22631.3155

**Reply by SpacedustPL** [reply]
I've created a simple bat file and added it into autostart, then it works well every time.

**Reply by daveo76** [reply]
I added 1 minute delay to it start up. Which made it work

**Reply by SpacedustPL** [reply]
Even 30 seconds is enough

**Comment by SpacedustPL** [comment]
I have the same issue on latest Windows 11 Pro 23H2 - build 22631.3155.

**Comment by OldManAlpha** [comment]
Also experiencing this on 21H2 19044.1288.
